shared/src/business/entities/cases/Case.ts

```diff
@@ -243,6 +243,39 @@ export class Case extends JoiValidationEntity {
     );
   }
 
+  static sortByDocketNumberAndGroupConsolidatedCases<
```

Comment:
There is a similar function [here](https://github.com/ustaxcourt/ef-cms/blob/d95d05f9ee81c6a2eb12ae73063c1ca9b693577c/shared/src/business/utilities/trialSession/getFormattedTrialSessionDetails.ts#L203), but 1) it isn't super intuitive and 2) it is wholly separate from our other sorting. I also keep running into this issue of "how, exactly, should I be sorting docket numbers in this context?"; my hope is that we can get better at naming our sorting functions and consolidating them.

Reply:
After this feature is delivered, I will remove the similar function and use this function instead. I would like to get this feature out before adding more work.

Reply:
Good points, agreed! I wonder if `Case.ts ` is the best place for this function? Not a big deal, but it's not single case specific, and this entity is already huge

Reply:
Eh, after reading further I see why this was added here. Maybe something for long-term consideration, but ignore the above comment for this PR

Reply:
Yes, I totally agree. What I would like to do is have a separate sorting file where we consolidate all of the case/docket sorting functions that seem somewhat scattered.

Reply:
I like that idea very much